buildsystem/spack/crusher/modules/dependencies.sh

```diff
@@ -22,21 +22,15 @@ module load cmake/3.20.6-clang-14.0.0-rocm5.2.0-mixed-lilujjv
 # blt@=0.4.1%clang@=14.0.0-rocm5.2.0-mixed build_system=generic arch=linux-sles15-zen3
 module load blt/0.4.1-clang-14.0.0-rocm5.2.0-mixed-pihruyq
 # hip@=5.2.0%clang@=14.0.0-rocm5.2.0-mixed~cuda+rocm build_system=cmake build_type=Release generator=make patches=959d1fe,c2ee21c arch=linux-sles15-zen3
-module load hip/5.2.0-clang-14.0.0-rocm5.2.0-mixed-26fjs6f
-# hsa-rocr-dev@=5.2.0%clang@=14.0.0-rocm5.2.0-mixed+image+shared build_system=cmake build_type=Release generator=make patches=9267179 arch=linux-sles15-zen3
-module load hsa-rocr-dev/5.2.0-clang-14.0.0-rocm5.2.0-mixed-44hnkkg
+module load hip/5.2.0-clang-14.0.0-rocm5.2.0-mixed-6ftqihk
+# hsa-rocr-dev@=5.2.0%clang@=14.0.0-rocm5.2.0-mixed~asan+image+shared build_system=cmake build_type=Release generator=make patches=9267179 arch=linux-sles15-zen3
```

Comment:
We need ROCm >= 5.6. Also, I suggest using Frontier modules rather than building our own.

Reply:
See #89 regarding rocm >=5.6.
For the `rocm` modules, I am using the system modules (see https://github.com/pnnl/ExaGO/blob/nicholson/frontier-eng145/buildsystem/spack/crusher/spack.yaml#L208), but Spack likes to add wrapper to it.